azext_iot/central/commands_destination.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List, Union
+from azext_iot.common import utility
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralDestinationProvider
+from azext_iot.central.models.enum import ApiVersion
+from azext_iot.central.models.v1_1_preview import (
+    DestinationV1_1_preview,
+    WebhookDestinationV1_1_preview,
+    AdxDestinationV1_1_preview,
+)
+
+
+def get_dataExport_destination(
+    cmd,
+    app_id: str,
+    destination_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
```

Comment:
What is the api_version for? Is there a list of available api versions (and what they correspond to)?

Reply:
Yes, there is a list of API version existed, currently we have preview, 1.1-preview and 1.0 three versions, this command only allowed with 1.1-preview.